src/V3Partition.cpp

```diff
@@ -2029,7 +2029,18 @@ class ThreadSchedule final {
     std::unordered_map<const ExecMTask*, uint32_t> threadId;
 
     // State for each mtask.
-    std::unordered_map<const ExecMTask*, MTaskState> mtaskState;
+    std::unordered_map<const ExecMTask*, MTaskState> mtaskStates;
+
+    MTaskState& mtaskState(const ExecMTask* mtaskp) { return mtaskStates[mtaskp]; }
+    const MTaskState& mtaskState(const ExecMTask* mtaskp) const { return mtaskStates.at(mtaskp); }
+    uint32_t mtaskThread(const ExecMTask* mtaskp) const {
```

Comment:
Suggest remove `threadId` map then rename this method to `threadId` for consistency

Reply:
I am talking about method `mtaskThread` that is.